tests/scripts/multi-node/build-rook.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+set -e
+
+
+#############
+# VARIABLES #
+#############
+
+rook_git_root=$(git rev-parse --show-toplevel)
+rook_kube_templates_dir="$rook_git_root/cluster/examples/kubernetes/"
+
+
+#############
+# FUNCTIONS #
+#############
+
+function fail_if {
+  if [[ "$(uname -n)" != "k8s-01" ]]; then
```

Comment:
does this function need to be updated since the build is on the host now?

Reply:
nop my bad, fixing this